Scripts/copy-products.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+set -e
```

Comment:
Nit: so that it's similar in all scripts:

```suggestion
set -e

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.
```

Reply:
Disagree here, copyright should be before any commands

Reply:
Update other places then)